### PR TITLE
Issue: Network replaced with Chain

### PR DESCRIPTION
**Pull request by ac9-xyz**
Replaced most instance of network to chain. There are fair amount of `.j2` files which I am unsure of the necessary changes or how it connects to {{ network }}

Please do a `ctl shift f` in the repo for network and let me know if there are further instances that need to be changed

### CODE REVIEW

File: roles/support_update_min_gas/tasks/main.yml [hunk 1]
```diff
@@ -10,7 +9,7 @@
 - name: Restart cosmovisor service
   become: true
   systemd:
-    name: '{{ network }}'
+    name: '{{ chain }}'
```
Comment (cosmos-spaces) [inline roles/support_update_min_gas/tasks/main.yml:12]: should be cosmovisor_service_name

File: vars/testnet/kyve.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-network: kyve
+chain: kyve
 folder: '.kyve'
 daemon: kyve
 chain_id: kyve-1
 node_version: 'v1.5.0'
-repo: https://github.com/KYVENetwork/chain.git
+repo: https://github.com/KYVEchain/chain.git
```
Comment (cosmos-spaces) [inline vars/testnet/kyve.yml:6]: Revert back

File: roles/support_price_feeder/templates/umee/mainnet.toml.j2 [hunk 3]
```diff
@@ -604,13 +604,13 @@ quote = "USDC"
 # provider_endpoints
 [[provider_endpoints]]
 name = "osmosis"
-rest = "https://api.osmo-api.prod.ojo.network"
-websocket = "api.osmo-api.prod.ojo.network"
+rest = "https://api.osmo-api.prod.ojo.chain"
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_price_feeder/templates/umee/mainnet.toml.j2]: Done.

File: roles/support_remove_node/tasks/main.yml [hunk 4]
```diff
@@ -1,15 +1,14 @@
----
 - name: Stop node process
   become: true
   systemd:
-    name: '{{ network }}'
+    name: '{{ chain }}'
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_remove_node/tasks/main.yml]: Updated.

File: roles/support_remove_node/tasks/main.yml [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
     state: stopped
     enabled: false
   ignore_errors: true
 
 - name: Remove project git folder
   file:
-    path: '{{ user_dir }}/{{ network }}'
+    path: '{{ user_dir }}/{{ chain }}'
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_remove_node/tasks/main.yml]: Done.

File: roles/support_remove_node/tasks/main.yml [hunk 6]
```diff
@@ -20,7 +19,7 @@
 - name: remove systemd file
   become: true
   file:
-    path: '/etc/systemd/system/{{ network }}.service'
+    path: '/etc/systemd/system/{{ chain }}.service'
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_remove_node/tasks/main.yml]: Changed.

File: roles/support_resync/templates/recover.sh.j2 [hunk 7]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-sudo service {{ network }} stop
+sudo service {{ chain }} stop
 sleep 2
 rm {{ user_dir }}/{{ folder }}/data -rf
 cp {{ user_dir }}/{{ folder }}/data-old {{ user_dir }}/{{ folder }}/data -r
-sudo service {{ network }} start
+sudo service {{ chain }} start
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_resync/templates/recover.sh.j2]: Changed.

File: roles/support_resync/templates/resync.sh.j2 [hunk 8]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-sudo service {{ network }} stop
+sudo service {{ chain }} stop
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_resync/templates/resync.sh.j2]: Switched.

File: roles/support_resync/templates/resync.sh.j2 [hunk 9]
```diff
@@ -21,5 +21,5 @@ mv {{ user_dir }}/{{ folder }}/config/addrbook.backup {{ user_dir }}/{{ folder }
 cp {{ user_dir }}/{{ folder }}/data-old/wasm {{ user_dir }}/{{ folder }}/data/wasm -r
 {% endif %}
 
-{{ user_dir }}/state_sync/{{ network }}.sh
-sudo service {{ network }} start
+{{ user_dir }}/state_sync/{{ chain }}.sh
+sudo service {{ chain }} start
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_resync/templates/resync.sh.j2]: Switched.

File: roles/support_snapshot/templates/snapshot.sh.j2 [hunk 10]
```diff
@@ -11,24 +11,24 @@
...
 # prune node
-sudo service {{ network }} stop
+sudo service {{ chain }} stop
 sleep 5
 {{ user_dir }}/go/bin/cosmprund prune ~/{{ folder }}/data
 sleep 5
 
 # Make sure the service is running
-sudo service {{ network }} start
+sudo service {{ chain }} start
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_snapshot/templates/snapshot.sh.j2]: Done.

File: roles/support_snapshot/templates/snapshot.sh.j2 [hunk 11]
```diff
@@ -11,24 +11,24 @@
...
 sleep 10
 
 # Get block height
 BLOCK_HEIGHT=$(curl -s http://localhost:{{ custom_port_prefix }}57/status | jq -r .result.sync_info.latest_block_height)
 
 # Stop service
-sudo service {{ network }} stop
+sudo service {{ chain }} stop
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_snapshot/templates/snapshot.sh.j2]: Done.

File: roles/support_snapshot/templates/snapshot.sh.j2 [hunk 12]
```diff
@@ -45,19 +45,19 @@ tar -cvf - data | lz4 > "{{ user_dir }}/$FILENAME"
 {% endif %}
 
 # Restart the service
-sudo service {{ network }} start
+sudo service {{ chain }} start
```
[the remark this answers is not in the document]
Comment (ac9-xyz) [reply roles/support_snapshot/templates/snapshot.sh.j2]: Done.

File: roles/support_update_min_gas/tasks/main.yml (hunk 1 above)
Comment (ac9-xyz) [reply roles/support_update_min_gas/tasks/main.yml]: Done.

**Comment by ac9-xyz** [comment]
Rebased for the the address issue fix and other items so that testing works well

**Review by cosmos-spaces — APPROVED** [review]

**Pull request merged**